Dodanie ogólnych plików i zdj

Sanjos55555 opened the pull request:
1.Dodałem mixina do vars
2.Dodałem zmienne do vars
3. Dodałem plik customer-reviews.html
4. Dodałem plik _customer-reviews.scss
5. Dodałem zdj do mojej sekcji.

Sanjos55555 committed:
Dodanie ogólnych plików i zdj

1.Dodałem mixina do vars
2.Dodałem zmienne do vars
3. Dodałem plik customer-reviews.html
4. Dodałem plik _customer-reviews.scss
5. Dodałem zdj do mojej sekcji.

QuickerMaths commented on src/sass/utils/_vars.scss:
```diff
@@ -22,3 +28,12 @@ $fs-h2-mob: 1.8rem;
 $fs-h3-mob: 3.4rem;
+
+//font-family,font-weight,font-size,line-height,letter-spacing
+@mixin ff-fw-fs-lh-ls($ff-raleway, $fw, $fs, $lh, $ls) {
```
jakbyś mógł zmienić nazwę mixina na np font-properties, bo tak jest niezbyt czytelnie

Sanjos55555 replied:
ok

Sanjos55555 committed:
Commit 2

Zmieniem nazwe mixina
zrobiem makiet w. mob
dodaem zmienną
poprawiłem ściezkę do plików.

QuickerMaths commented on src/customer-reviews.html:
```diff
@@ -0,0 +1,43 @@
+        </p>
+      </div>
+
+      <div class="container-reviews__points-block">
```
zamiast divów w tel liście powinieneś użyć tagów ul i  li

Sanjos55555 committed:
commit 3

-wersja mobilna bez swipera
-zmieniełem px na rem
-dodałem brakujące zdj @x3
-przeniosłem pliku do odpowiednich folderów
-fonty nie działają, bo nie robiłem jeszcze pool.

QuickerMaths committed:
Merge branch 'main' into customer-reviews-/-Oleksandr-Zakharchenko
